go/database/log/migrations/20240401221053_initial.sql

```diff
@@ -0,0 +1,15 @@
+-- Create "collection" table
+CREATE TABLE "public"."collection" (
+  "id" text NOT NULL,
+  "record_compaction_offset_position" bigint NOT NULL,
```

Comment:
Can you comment on the meaning of this field?

Reply:
100%, will do.